projects/bp-gallery/src/helpers/app-helpers.ts

```diff
@@ -56,15 +56,41 @@ export const asUploadPath = (media: FlatUploadFile | undefined, options: UploadO
  */
 export const buildHttpLink = (
   token: string | null,
-  openAlert?: (alertOptions: AlertOptions) => void
+  openAlert?: (alertOptions: AlertOptions) => void,
+  anonymousId?: string | null
 ) => {
   let httpLink = createHttpLink({
     uri: `${apiBase}/graphql`,
     headers: {
+      'Access-Control-Request-Headers': 'anonymousId',
       authorization: token ? `Bearer ${token}` : '',
+      anonymousId: anonymousId,
     },
   });
 
+  const growthbookLink = new ApolloLink((operation, forward) => {
+    return forward(operation).map(response => {
+      const experimentData = JSON.parse(
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-call
+        operation.getContext().response.headers.get('x-growthbook')
+      );
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+      experimentData?.map((experiment: { experimentId: string; variationId: string }) => {
```

Comment:
since you don't use the result, I think `forEach` is more appropriate than `map`